src/features/validation/validate.ts

```diff
@@ -168,6 +171,24 @@ export class ApplyMarkersCommand extends FeedbackCommand {
     }
 }
 
+function addCSSClassToIssueParent(modelElement: SParentElement, issueMarker: SIssueMarker) {
+    if (modelElement.cssClasses === undefined) {
+        modelElement.cssClasses = [getSeverity(issueMarker)];
+    } else {
+        modelElement.cssClasses.push(getSeverity(issueMarker));
+    }
+}
+
+function removeCSSClassFromIssueParent(modelElement: SParentElement, issueMarker: SIssueMarker) {
```

Comment:
Please use the utility method from `src/utils/smodel-utils.ts` for this

Reply:
Thanks, addressed in a follow-up commit